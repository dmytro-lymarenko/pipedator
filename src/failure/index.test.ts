import { failure } from './index';

describe('optional()', () => {
	it('should fail', () => {
		expect(failure().validate(1)).toMatchSnapshot();
	});

	it('should support custom message', () => {
		expect(failure('Custom message').validate(null)).toMatchSnapshot();
	});
});
