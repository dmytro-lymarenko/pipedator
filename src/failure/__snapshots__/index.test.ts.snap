// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`failure() should fail 1`] = `
Object {
  "message": "Should always fail",
  "path": Array [],
  "type": "single",
  "value": 1,
}
`;

exports[`failure() should support custom message 1`] = `
Object {
  "message": "Custom message",
  "path": Array [],
  "type": "single",
  "value": null,
}
`;
